tdesign/src/Upload/Upload.tsx

```diff
@@ -0,0 +1,136 @@
+import { useCallback, useMemo, CSSProperties } from "react";
+import { useDropzone, FileWithPath } from "react-dropzone";
+import { Box, Typography } from "@material-ui/core";
+import type { BoxProps } from "@material-ui/core";
+import { ArrowForward } from "@material-ui/icons";
+import { Button, LinkButton } from "../Button";
+import UploadCloudIcon from "./UploadCloudIcon";
+// import { upload } from "./uploadHelper";
+
+interface IUploadProps extends BoxProps {
+  uploadHandler?: () => {};
+  maxSizeMB?: number;
+  maxFiles?: number;
+  accept?: string;
+}
+
+const Upload = ({
+  sx = {},
+  maxSizeMB = 100,
+  maxFiles = 8,
+  accept,
+  ...rest
+}: IUploadProps) => {
+  const onDrop = useCallback((acceptedFiles) => {
+    acceptedFiles.forEach((file) => {
+      const reader = new FileReader();
+
+      reader.onabort = () => console.log("file reading was aborted");
+      reader.onerror = () => console.log("file reading has failed");
+      reader.onload = () => {
+        const binaryStr = reader.result;
+        console.log(binaryStr);
+      };
+      reader.readAsArrayBuffer(file);
```

Comment:
Is there a reason we need to read the contents of the (potentially large) file into memory? AFAIU I don't think this is necessary to simply upload the file without reading its contents